app/components/post_body/post_body.js

```diff
@@ -104,6 +118,10 @@ export default class PostBody extends PureComponent {
                 isLongPost: true,
             });
         }
+
+        if (this.props.isLastPost) {
+            this.logTelemetry();
```

Comment:
I don't like doing this in the Post component since those can show up outside of the main view. It definitely doesn't belong in `measurePost` since that should just be for actually measuring the post.

Perhaps we could find some way to do this in the Channel screen's `componentDidUpdate` or `componentDidMount'`? That's around where I'd expect to find it.

Reply:
That (channel screen) was the initial location however it's giving shorter time period which is not realistic.  Parent component may already be rendered but the grandchildren are not yet.
In this place is where I'm getting a closer time period to end rendering.  The idea is to place this on the leaf component as much as possible where it can measure activities like image loading, etc.

Reply:
That makes sense. I'm worried that it'll make it hard to isolate this logic, but it's probably fine.